src/containers/Categories/Categories.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import Category from "../../components/Category/Category";
+import "./categories.css";
+
+class Categories extends Component {
+  state = {
+    categories: [
```

Comment:
Testing data should be moved to separate file in same folder, after this will be connected to real API, the file can be deleted.